packages/ember-cli-code-coverage/lib/gjs-gts-istanbul-ignore-template-plugin.js

```diff
@@ -0,0 +1,47 @@
+const gjsGtsTemplateIgnoreVisitor = {
+  CallExpression(path) {
+    const { node } = path;
+    let { callee } = node;
+
+    if (callee.name !== 'template') {
+      return;
+    }
+
+    const callScopeBinding = path.scope.getBinding('template');
+
+    if (
+      callScopeBinding.kind === 'module' &&
+      callScopeBinding.path.parent.source.value === '@ember/template-compiler'
+    ) {
+      const babelIgnoreComment = {
+        type: 'CommentBlock',
+        value: ' istanbul ignore next ',
+      };
+
+      if (!path.findParent((path) => path.isStatement()).node.leadingComments) {
+        path.findParent((path) => path.isStatement()).node.leadingComments = [];
+      }
+      path
+        .findParent((path) => path.isStatement())
+        .node.leadingComments.push(babelIgnoreComment);
+    }
+  },
+};
+
+module.exports = function () {
+  return {
+    visitor: {
+      Program: {
+        enter(path, state) {
+          const filename = state.file.opts.filename;
+
+          if (!filename?.match(/\.g[tj]s$/)) {
+            return;
+          }
+
+          path.traverse(gjsGtsTemplateIgnoreVisitor, state); // we need to do early traverse to make sure this runs before istanbuls plugin
```

Comment:
```suggestion
          // we need to do early traverse to make sure this runs before istanbuls plugin
          path.traverse(gjsGtsTemplateIgnoreVisitor, state);
```
just a tiny bit more readable

Reply:
Fixed @SergeAstapov